config/dev.exs

```diff
@@ -1,8 +1,8 @@
 use Mix.Config
 
 config :tic_tac_toe, :deps, %{
-  input: Input,
-  output: Output,
-  validity: Validity,
-  human_player: HumanPlayer
+  validation: Validation,
```

Comment:
mentioned this IRL but I'm not sure if these are really deps if they live inside the project. Maybe we can get away with only configuring `:console_io`?